twake/backend/node/src/services/messages/entities/messages.search.ts

```diff
@@ -24,9 +26,11 @@ export default {
   esMapping: {
     properties: {
       text: { type: "text" },
+      user_id: { type: "text" },
```

Comment:
If you want to search using this format:
`...(options.sender ? { $in: [["user_id", [options.sender]]] } : {}),`
You need to use this type:
`user_id: { type: "keyword" }`

Reply:
hm… but it works this way…
ok, I'll change

Reply:
I didn't know it worked haha :p 
But another point for me is that everything you put in "text" will be searchable in the "query" like if the message text is "I love the option b514" and sender is "0ac1d15a-a5dd-11ec-b514-476f788a5663" then searching for the text "b514" will return all the messages from this sender.
So change it to `keyword` anyway ;)